documentation/docs/get-started/llm-studio-flow.md

```diff
@@ -1 +1,45 @@
-# [application_name] flow
+# Experiment (model) flow
+
+The flow of creating and fine-tuning large language models using H2O LLM Studio can be summarized in the following sequential steps:
+
+- [Step 1: Import a dataset](#step-1-import-a-dataset)
+- [Step 2: Create an experiment](#)
+- [Step 3: Monitor an experiment](#)
+- [Step 4: Compare experiments](#)
+- [Step 5: Export a model to Hugging Face Hub](#)
+
+In the below sections, each step above, in turn, is summarized.
+
+## Step 1: Import a dataset
+
+As the first step in the experiment flow, import your dataset to H2O LLM Studio. 
+
+- To learn about supported data connectors and data format, see [Supported data connectors and format](../guide/data-connectors-format).
+- To learn about how to import a dataset to H2O LLM Studio, see [Import a dataset](../guide/import-dataset).
+- To learn about reviewing and editing a dataset, see [View and manage dataset](../guide/view-dataset.md)
+
+## Step 2: Create an experiment
+
+As the second step in the experiment flow, create an experiment using the imported dataset. H2O LLM Studio offers several hyperparameter settings that you can adjust for your experiment model. To ensure that your training process is effective, you may need to specify the hyperparameters like learning rate, batch size, and the number of epochs. H2O LLM Studio provides an overview of all the parameters you’ll need to specify for your experiment.
+
+- To learn about the settings available for creating an experiment, see [Experiment settings](../guide/experiments/experiment-settings.md).
+- To learn about creating a new experiment, see [Create an experiment](../guide/experiments/create-an-experiment.md)
```

Comment:
I would switch around the order that this is listed in. First let's direct them to creating an experiment. Experiment settings seems more like a reference doc for more info. WDYT?

Reply:
Yeah, that makes sense. Thank you.